app/components/admin/table_actions_component.html.erb

```diff
@@ -2,10 +2,10 @@
   <%= content %>
 
   <% if actions.include?(:edit) %>
-    <%= action(:edit, **edit_options.merge(text: edit_text, path: edit_path)) %>
+    <%= action(:edit, text: edit_text, path: edit_path, **edit_options) %>
```

Comment:
With this change, the code would be like `**({ text: edit_text, path: edit_path }.merge(edit_options))`, which is the opposite of what we had :thinking:.

```suggestion
    <%= action(:edit, **edit_options, text: edit_text, path: edit_path) %>
```